go.mod

```diff
@@ -22,5 +22,5 @@ require (
 	// NOTE: urfave/cli must be <= v1.22.1 due to a regression: https://github.com/urfave/cli/issues/1092
 	github.com/urfave/cli v1.22.1
 	github.com/vishvananda/netlink v1.1.0
-	golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1
+	golang.org/x/sys v0.0.0-20200909081042-eff7692f9009
```

Comment:
note that the golang.org/x/sys updates are incompatible with the containerd/console package, and break compile on macOS. A fix is open for this in https://github.com/containerd/console/pull/40

I don't think anything in runc is compiled on macOS currently, so probably not a blocker, but it may cause issues for projects that use runc as a go module, as it will force them to update to a newer golang.org/x/sys version